src/sonic-config-engine/tests/sample_output/l2switch.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
we need management port configuration.

Reply:
This PR focuses on hwsku related feature.

---
In reply to: [183607010](https://github.com/Azure/sonic-buildimage/pull/1631#discussion_r183607010) [](ancestors = 183607010)